File: Application/app/src/main/java/com/example/sage/foodsharebook/models/LoginResponse.java
package com.example.sage.foodsharebook.models;

import com.google.gson.annotations.SerializedName;

public class LoginResponse{

	@SerializedName("auth_token")
	private String authToken;

	@SerializedName("user_id")
	private int user_id;

	public void setAuthToken(String authToken){
		this.authToken = authToken;
	}

	public String getAuthToken(){
		return authToken;
	}

	public int getUser_id() {
		return user_id;
	}

	public void setUser_id(int user_id) {
		this.user_id = user_id;
	}
}

File: Application/app/src/main/java/com/example/sage/foodsharebook/models/LoginResponse.kt
package com.example.sage.foodsharebook.models

import com.google.gson.annotations.SerializedName

class LoginResponse {

    @SerializedName("auth_token")
    var authToken: String? = null

    @SerializedName("user_id")
    var user_id: Int = 0
}